Lib/Src/CDispatcher.cpp

```diff
@@ -133,54 +133,34 @@ void CDispatcher::run()
 /**
  * @brief Method to process incoming commands.
  *
- * @param command Command that arrived via coms channel
+ * @param command ICommand pointer
```

Comment:
The header should add meaning, so it'd be better if this read: Pointer to a command object via an interface class.

Reply:
Fixed